pkg/scheduler/apis/config/testing/compatibility_test.go

```diff
@@ -1794,3 +1795,284 @@ func TestPluginsConfigurationCompatibility(t *testing.T) {
 		})
 	}
 }
+
+func Test(t *testing.T) {
```

Comment:
Algorithm provider is going away very soon. Remove this.

Reply:
It's an error, I have removed this.